dashboard/src/features/shared/components/BaseList/EmptyList.jsx

```diff
@@ -13,28 +13,17 @@ class EmptyList extends React.Component {
     if (!this.props.loadedOnce) {
       emptyBlock = <span>LOADING…</span>
     } else if (this.props.showFirstTimeFlow && this.props.skipCreate) {
-      emptyBlock = <div>
-        <span className={styles.emptyLabel}>
-          There are no {this.props.objectName.toLowerCase()}s on the blockchain
-        </span>
-      </div>
+      emptyBlock = <div>{this.props.emptyContent}</div>
     } else if (this.props.showFirstTimeFlow) {
       emptyBlock = <div>
-        <span className={styles.emptyLabel}>Create your first {this.props.objectName}</span>
+        {this.props.emptyContent}
         {this.props.newButton}
       </div>
     } else if (!this.props.showFirstTimeFlow) {
       classNames.push(styles.noResults)
-      emptyBlock = <div>
+      emptyBlock = <div className={styles.emptyContainer}>
         <span className={`${styles.emptyLabel} ${styles.noResultsLabel}`}>No results for query:</span>
         <code className={styles.code}>{this.props.currentFilter.filter}</code>
-        <div className={styles.queryHelp}>
```

Comment:
I might have missed a discussion, but why are we removing the doc links from this branch? It's a [relatively new addition](https://github.com/chain/chain/pull/223).